react-app/pomodoro-timer-api/db.js

```diff
@@ -1,15 +1,14 @@
-const {Client} = require("pg")
-const {getDatabaseUri} = require("./config")
+const { Client } = require("pg")
+const { getDatabaseUri } = require("./config")
 require("colors")
 
-const db = new Client({connectionString:getDatabaseUri()})
+const db = new Client({ connectionString:getDatabaseUri() })
 
-db.connect((err)=>{
-    if (err){
-        console.error("connection error".red, err.stack)
-    }
-    else{
-        console.log("successfully connected to postgres db!".green)
+db.connect((err) => {
+    if (err) {
+        console.error("connection error", err.stack)
+    } else {
+        console.log("successfully connected to postgres db!".blue)
```

Comment:
found a console log

Reply:
that console.log has been there since we start production a couple weeks ago. I think it's meant to ensure that we connect to the database successfully. Do we really have to remove it?

Reply:
oooh, gotcha. Ok, my bad, G2G

Reply:
@Rebecca-CYLi